### ISSUE TITLE
Implementação script

### ISSUE THREAD

**Issue by hahndouglasf** [marceloengecom/Odoo-Install#2]
Olá, gostaria primeiramente de agradecer pelo seu script.

Gostaria de saber se há como implementar o script para a instalação automatizada na nova versão do Odoo com os módulos do diretório " https://github.com/OCA/l10n-brazil "

Grato!

**Comment by marceloengecom** [comment]
Sim...é possível, mas estou sem tempo para fazer essa alteração no script nesse momento. Agora, você pode escolher os componentes principais a instalar, por exemplo, não instale os módulos da TrusCode e instale os módulos da OCA/l10n-brazil
